dynawo/sources/Models/CPP/ModelNetwork/DYNModelTapChanger.h

```diff
@@ -51,114 +59,115 @@ class ModelTapChanger {
    * @brief  return the name of the tap changer
    * @return name of the tap changer
    */
-  inline std::string id() const {
-    return id_;
-  }
+  std::string const& id() const { return id_; }
```

Comment:
Please put back the explicit inline to make sure to have an inlining request for any compiler

Reply:
To be consistent with the remaining of the code: we also usually write const Object&